frontend/src/app/sessions/user-sessions-wrapper/active-sessions/connection-dialog/connection-dialog.component.html

```diff
@@ -18,32 +18,11 @@
     You can always return to this dialog by clicking on "Connect" in the "Active
     sessions" overview. <br />
     <div *ngIf="session?.t4c_password">
-      <span
-        class="cursor-copy border font-mono"
-        [cdkCopyToClipboard]="session!.t4c_password"
-        (click)="showClipboardMessage()"
-      >
-        <span [ngClass]="{ blur: !t4cPasswordRevealed }">{{
-          session!.t4c_password
-        }}</span></span
-      >
-
-      <button
-        *ngIf="!t4cPasswordRevealed"
-        (click)="this.t4cPasswordRevealed = true"
-        mat-mini-fab
-        matTooltip="Show session token"
-      >
-        <mat-icon> blur_off</mat-icon>
-      </button>
-      <button
-        *ngIf="t4cPasswordRevealed"
-        (click)="this.t4cPasswordRevealed = false"
-        mat-mini-fab
-        matTooltip="Hide session token"
-      >
-        <mat-icon> blur_on</mat-icon>
-      </button>
+      <app-display-value
+        [value]="session!.t4c_password"
+        [blurValue]="true"
+        valueName="TeamForCapella token"
```

Comment:
is it `TeamForCapella` or `Team4Capella` ? Or do we use the second just sometimes in the code?

Reply:
`T4C`, `TeamForCapella`, `Team for Capella` and `Team4Capella` can be used as synonyms.